content/applications/finance/accounting/fiscal_localizations/localizations/luxembourg.rst

```diff
@@ -8,9 +8,21 @@ Configuration
 :ref:`Install <general/install>` the following modules to get all the features of the Luxembourgish
```

Comment:
```suggestion
:ref:`Install <general/install>` the following modules to get all features of the Luxembourgish
```

Reply:
Ah okay, good to know. It's not part of this commit and this text is already in several docs in several versions so I'll remove this "the" with another PR. thanks :)

Reply:
@toaa-odoo